.vitepress/config.js

```diff
@@ -46,6 +46,7 @@ export default defineConfig({
 			{
 				text: '2.4',
 				items: [
+					{ text: '3.0', link: 'https://doc.dovecotpro.org/3.0/' },
```

Comment:
I would suggest label to be "3.0 (Pro)" or "Pro (3.0)" to distinguish between CE docs